### PR TITLE
Use emoji in ActionColumn

### PR DESCRIPTION
**Pull request by np25071984**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #16359

### CODE REVIEW

**Comment by GHopperMSK** [comment]
image

File: framework/grid/ActionColumn.php [hunk 1]
```diff
@@ -142,9 +142,9 @@ public function init()
      */
     protected function initDefaultButtons()
     {
-        $this->initDefaultButton('view', 'eye-open');
-        $this->initDefaultButton('update', 'pencil');
-        $this->initDefaultButton('delete', 'trash', [
+        $this->initDefaultButton('view', '👁');
+        $this->initDefaultButton('update', '✏');
+        $this->initDefaultButton('delete', '🗑', [
```
Comment (samdark) [inline framework/grid/ActionColumn.php]: I'd use ❌, ✎, 👁

Comment (samdark) [reply framework/grid/ActionColumn.php]: https://github.com/yiisoft/yii2/issues/16359#issuecomment-403242885

Comment (rob006) [reply framework/grid/ActionColumn.php]: I would leave 🗑 - it is more similar to previous icon and IMO fits better for delete action.

Comment (samdark) [reply framework/grid/ActionColumn.php]: OK.

**Comment by samdark** [comment]
Tests should be adjusted: https://travis-ci.org/yiisoft/yii2/jobs/477889325

**Comment by samdark** [comment]
Oh! The pull request made into wrong repository. The change was not planned for 2.0...

**Comment by samdark** [comment]
I've moved the issue.

**Comment by samdark** [comment]
Thanks for doing it, will be easier to proceed doing the change for 3.0.

**Pull request closed without merging**